prow/config.yaml

```diff
@@ -134,6 +134,58 @@ presubmits:
           secret:
             defaultMode: 256
             secretName: ssh-key-secret
+    - name: pull-kubernetes-e2e-kubeadm-gce
+      agent: kubernetes
+      context: pull-kubernetes-e2e-kubeadm-gce
+      always_run: false
+      max_concurrency: 8
+      skip_report: true
+      rerun_command: "/test pull-kubernetes-e2e-kubeadm-gce"
+      trigger: "(?m)^/test pull-kubernetes-e2e-kubeadm-gce,?(\\s+|$)"
```

Comment:
Not sure how to make this run only on `cmd/kubeadm` jobs when this is a post-bazel job. Pointers would be appreciated